service/geopmdpy/service.py

```diff
@@ -57,6 +59,20 @@ def __init__(self):
             if write_pid is not None and not self._active_sessions.is_client_active(write_pid):
                 self._close_session_write(lock, write_pid)
 
+    def _check_plugins(self):
+        # Similar logic to geopm_plugin.cpp -> plugin_load()
+        so_suffix = '.so.' + version.__abi_version__.replace(':', '.')
+        plugin_path = os.getenv('GEOPM_PLUGIN_PATH')
+        if plugin_path is not None:
+            user_paths = plugin_path.split(':')
+            for path in user_paths:
+                if system_files.is_secure_path(path):
+                    for file_path in glob.glob(os.path.join(path, '*')):
```

Comment:
at least currently we also have .json files in the plugin directory.  We don't want to error when these .json files don't end in .so.1.0.0.